buildSrc/build.gradle.kts

```diff
@@ -20,6 +20,7 @@ dependencies {
     implementation("com.android.tools.build:gradle:_")
     implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:_")
     implementation("org.jetbrains.kotlin:kotlin-compiler-embeddable:_")
+    implementation("org.jetbrains.dokka:dokka-gradle-plugin:_")
```

Comment:
Is that required? I want to use the plugins DSL over manual buildscript classpath dependency setting whenever possible.

Reply:
Yeah, the precompiled plugin needs access to the Dokka code, if you don't add it, you can't reference dokka code in precompiled plugins, like when you use core module from androidx in your project.

Reply:
So, precompiled Gradle scripts don't really support the plugins DSL?

Reply:
Precompiled plugins can't set the version, I don't know what do you mean with DSL, but because you are not able to set a version, if you don't define it in the dependencies block, you will not be able to use the plugin.

Reply:
Check my last commit, it is related to what you say about DSL?